refactor/ast.py

```diff
@@ -51,6 +52,32 @@ def apply_indentation(
         if len(self.data) >= 1:
             self.data[-1] += str(end_suffix)  # type: ignore
 
+    def apply_indentation_from_source(
```

Comment:
Does this mean we can also get rid of the `apply_indentation` method? I know we use it on InsertAfter which works a bit different but ideally these two are very similar on how they are written. Maybe it makes sense to unify them both?

Reply:
Possibly, I did not want to break anything, adding a duplicate for "my purpose" seemed the better route. I will look into it